engine/app/components/citizens_advice_components/error_summary.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= tag.div(
+  class: "cads-error-summary js-error-summary",
+  "data-testid": "error-summary",
+  "aria-labelledby": "error-summary-title",
+  "aria-live": "assertive",
+  role: "alert",
+  tabindex: "0",
+  autofocus: autofocus?
+) do %>
+  <%= content_tag(
```

Comment:
Definitely not legacy: `content_tag` accepts the name of the tag as a symbol where as `tag` is intended to be used as a named method call `e.g. tag.h1`. Because we are dynamically generating the tag name to use based on the heading level we want `content_tag` here.